README.md

```diff
@@ -1,2 +1,175 @@
-# web-file-upload
-A file management system built for vue and react that allows for single and multiple file uploading with a preview feature.
+# Web File Management with Preview - Fully Customized
+
+A versatile and user-friendly file management system built for vue and react that allows for single and multiple file uploading with a preview feature. It allows you to select files and preview them, returning an array of selected files. It allows customizing design by overriding the style classes.
+
+<img src="./gifs/full.gif"/>
+
+---
+
+### This repository contains two component library, one for Vue and one for React. Each has its own set of features and information. Explore the individual folders for more details.
```

Comment:
The library provides support for Vue and React. Each has its own set of features and information. Explore the individual folders for more details.